pkg/client/client.go

```diff
@@ -280,7 +282,25 @@ func (c *Client) DeleteDeployment(name string) (*common.Deployment, error) {
 	return deployment, nil
 }
 
-// PostDeployment posts a deployment objec to the manager service.
-func (c *Client) PostDeployment(cfg *common.Configuration) error {
-	return c.CallService("/deployments", "POST", "post deployment", cfg, nil)
+// PostDeployment posts a deployment object to the manager service.
+func (c *Client) PostDeployment(name string, cfg *common.Configuration) error {
+	d, err := yaml.Marshal(cfg)
+	if err != nil {
+		return err
+	}
+	// This is a stop-gap until we get this API cleaned up.
+	t := common.Template{
+		Name:    name,
+		Content: string(d),
+	}
+
+	data, err := json.Marshal(t)
+	if err != nil {
+		return err
+	}
+
+	var out interface{}
```

Comment:
You could also use an anonymous stuct here.  Save some mem points too!
`var out struct{}`